Cargo.toml

```diff
@@ -5,44 +5,39 @@ authors = ["The Attestation Agent Authors"]
 publish = false
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-futures = "0.3.5"
+aes-gcm = { version = "0.10.1", optional = true }
+anyhow = "1.0"
+async-trait = "0.1.56"
+base64 = "0.13.0"
+bincode = { version = "1.3.3", optional = true }
+foreign-types = { version = "0.5.0", optional = true }
+log = { version = "0.4.14", optional = true }
+openssl = { version = "0.10", features = ["vendored"], optional = true}
+prost = { version = "0.11.0", optional = true }
+rand = { version = "0.8.5", optional = true }
+rsa = { version = "0.6.1", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-aes-gcm = "0.10.1"
-base64 = "0.13.0"
-anyhow = "1.0"
-log = "0.4.14"
-env_logger = "0.9.0"
-foreign-types = "0.5.0"
-openssl = { version = "0.10", optional = true, features = ["vendored"]}
+sha2 = { version = "0.10", optional = true }
 strum = { version = "0.24.0", features = ["derive"] }
-rsa = "0.6.1"
-sha2 = "0.10"
-rand = "0.8.5"
-async-trait = "0.1.56"
+tdx-attest-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", optional = true }
 tonic = { version = "0.8.0", optional = true }
-prost = { version = "0.11.0", optional = true }
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"], optional = true }
 uuid = { version = "1.1.2", features = ["serde", "v4"], optional = true }
-bincode = { version = "1.3.3", optional = true }
-futures-util = "0.3.21"
-tdx-attest-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", optional = true }
 
 [dev-dependencies]
 tokio = { version = "1.20.1", features = ["macros", "rt-multi-thread"] }
 
 [build-dependencies]
-tonic-build = "0.8.0"
-shadow-rs = "0.16.1"
+tonic-build = { version = "0.8.0", optional = true }
 
 [features]
 default = ["sample_kbc"]
-sample_kbc = []
-cc_kbc = ["tdx-attest-rs"]
-eaa_kbc = []
+sample_kbc = ["aes-gcm"]
+cc_kbc = ["aes-gcm", "rand", "rsa", "sha2", "tdx-attest-rs"]
```

Comment:
it seems like this uses `log` too, and in general, I think `log` should be used in all KBCs. So maybe not mark it as optional?

Reply:
yeah, `log` is popular enough and it deserves "mandatory", so removed "optional" here.